style.css

```diff
@@ -75,18 +91,68 @@
   padding: 25px 142px;
 }
 
-.card {
+.height-100vh {
   height: calc(100vh - 10vh);
 }
 
-.card-title {
+.width-920 {
   width: 920px;
 }
 
+.row {
+  display: grid;
+  grid-template-rows: auto;
+  row-gap: 50%;
+}
+
+.card {
+  background-color: #fff;
+  padding: 16px 16px 24px 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-sub-title {
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 80%;
+}
+
+.card-title {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.canopy {
+  font-size: 13px;
+  line-height: 16px;
+  color: #7ab69a;
+  margin: 20px 0;
+  width: 85%;
+}
```

Comment:
- [ ] Please ensure that the text color matches figma design, working perfectly with given template always shows our capability to deliver whatsoever is expected of us. Thanks 🙏 .

## Your Design
![ttaaa](https://user-images.githubusercontent.com/103386183/205292809-99bb04cd-3583-4e30-ac53-5e343fac00f2.png)